symfony/framework-bundle/3.3/manifest.json

```diff
@@ -18,6 +18,7 @@
     },
     "gitignore": [
         ".env",
+        "/config/services_*.yaml",
```

Comment:
No sure why these file should be ignored :/ they are part of the repository, no?